packages/vulkan_tools.rb

```diff
@@ -6,21 +6,19 @@
 class Vulkan_tools < Package
   description 'Vulkan Utilities and Tools'
   homepage 'https://www.khronos.org/vulkan/'
-  version '1.3.251'
+  version '1.3.257'
+  compatibility 'all'
   license 'custom'
-  compatibility 'x86_64 aarch64 armv7l'
   source_url 'https://github.com/KhronosGroup/Vulkan-Tools.git'
   git_hashtag "v#{version}"
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_tools/1.3.251_armv7l/vulkan_tools-1.3.251-chromeos-armv7l.tar.zst',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_tools/1.3.251_armv7l/vulkan_tools-1.3.251-chromeos-armv7l.tar.zst',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_tools/1.3.251_x86_64/vulkan_tools-1.3.251-chromeos-x86_64.tar.zst'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_tools/1.3.257_armv7l/vulkan_tools-1.3.257-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_tools/1.3.257_armv7l/vulkan_tools-1.3.257-chromeos-armv7l.tar.zst'
```

Comment:
Missing i686 and x86_64 binaries?

Reply:
Fixed! But there are no longer i686 binaries...